src/io/iologindata.cpp

```diff
@@ -83,22 +83,22 @@ void IOLoginData::updateOnlineStatus(uint32_t guid, bool login) {
 	Database::getInstance().executeQuery(query.str());
 }
 
-// The boolean "disable" will desactivate the loading of information that is not relevant to the preload, for example, forge, bosstiary, etc. None of this we need to access if the player is offline
-bool IOLoginData::loadPlayerById(std::shared_ptr<Player> player, uint32_t id, bool disable /* = true*/) {
+// The boolean "disableOfflineUnrelevantInformations" will desactivate the loading of information that is not relevant to the preload, for example, forge, bosstiary, etc. None of this we need to access if the player is offline
```

Comment:
This could be changed to "disableOfflineIrrelevantInformation"
We have 2 typos fixes in this variable: Unrelevant to Irrelevant  and Informations to Information